includes/admin/class-sensei-learners-main.php

```diff
@@ -460,7 +461,7 @@ protected function get_row_data( $item ) {
 					$reset_label  = esc_html__( 'Remove progress', 'sensei-lms' );
 				}
 
-				$actions[] = '<a class="learner-async-action button" data-user-id="' . esc_attr( $user_activity->user_id ) . '" data-action="' . esc_attr( $reset_action ) . '" data-post-id="' . esc_attr( $post_id ) . '" data-post-type="' . esc_attr( $post_type ) . '">' . $reset_label . '</a>';
+				$row_actions[] = '<span class="delete"><a class="learner-async-action delete" data-user-id="' . esc_attr( $user_activity->user_id ) . '" data-action="' . esc_attr( $reset_action ) . '" data-post-id="' . esc_attr( $post_id ) . '" data-post-type="' . esc_attr( $post_type ) . '">' . $reset_label . '</a></span>';
```

Comment:
Is there a way to break up these long lines?

Reply:
Updated in this method: https://github.com/Automattic/sensei/pull/3404/commits/9c4be1647972340660e7b7f588b4b88392667d3f